docs/en/tutorials/distribute_train.md

```diff
@@ -119,3 +135,105 @@ done
 When training other models, simply replace the yaml config file path in the script, i.e. `path/to/model_config.yaml`.
 
 After the training has started, and you can find the training log `train.log` in the project root directory.
+
+#### 1.2.2 Running on Four (Partial) Devices
+
+To run training on four devices, for example, `{4, 5, 6, 7}`, the `RANK_TABLE_FILE` and the run script are different from those for running on eight devices.
```

Comment:
Can we start with any device, e.g. 1?
```shell
python hccl_tools.py --device_num "[1, 3)"
```

Reply:
yes, I have tried training on {1,2}. It can work.